lib/styleProcessors/index.ts

```diff
@@ -0,0 +1,58 @@
+import merge from "merge-source-map";
+
+export interface StylePreprocessor {
+  render(
+    source: string,
+    map: any | null,
+    options: any
+  ): StylePreprocessorResults;
+}
+
+export interface StylePreprocessorResults {
+  source: string;
+  map?: any;
```

Comment:
There are a bunch of semicolons, make sure to stay consistent with the style of the rest of the codebase